serializer/src/main/scala-2.13/org/virtuslab/akkasaferserializer/RuntimeReflections.scala

```diff
@@ -0,0 +1,15 @@
+package org.virtuslab.akkasaferserializer
+
+import org.reflections8.Reflections
+
+import scala.jdk.CollectionConverters._
```

Comment:
Nit: there is an import that's portable b/w both Scala 2.12 and 2.13 (although deprecated in the latter)... `import scala.collection.JavaConverters._`?

Reply:
So duplicating code b/w scala-2.12 and scala-2.13 can be avoided

Reply:
```
[error] [...]\RuntimeReflections.scala:14:86: object ImplicitConversions in package convert is deprecated (since 2.13.0): Use `scala.jdk.CollectionConverters` instead
[error]   def findAllObjects[T](cl: Class[T]): Seq[Class[_ <: T]] = reflections.getSubTypesOf(cl).toSeq
[error]                                                                                      ^
[error] one error found
[error] (serializer / Compile / compileIncremental) Compilation failed
```
Deprecated but doesn't compile

Reply:
Ok, found it.
It's becaouse of flag `-Xfatal-warnings`.
Should we remove it?

Reply:
Ouch, right... remove `-deprecation` maybe, if it doesn't help, then leave as is

Reply:
No luck. It seems that's the way it's supposed to work.